dts/bindings/clock/st,stm32-clock-mco.yaml

```
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024, Joakim Andersson
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+compatible: "st,stm32-clock-mco"
+
+description: |
+        STM32 Microcontroller clock output (MCO)
+        Configure the microcontroller clock output pin and additional properties,
+        e.g. clock source and prescaler.
+        Example:
+                &mco1 {
+                        clocks = <&rcc STM32_SRC_LSE MCO1_SEL(7)>;
+                        prescaler = <MCO1_DIV(1)>;
```

To be consistent with the macros provided in headers:
```suggestion
                        prescaler = <MCO1_PRE(1)>;
```